test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -544,7 +545,7 @@ class RepresenteeControllerSpec
               session,
               journey.ggCredId,
               "representee.enterName.personalRep.title",
-              routes.RepresenteeController.isFirstReturn(),
+              triageRoutes.CommonTriageQuestionsController.furtherReturnHelp(),
```

Comment:
it'd be good to also have a test in the cya section of the spec to test the new case you've introduced - that holds for this spec and the single and multiple triage controller specs as well